logger.js

```diff
@@ -62,10 +62,8 @@ const log = (data, level, context) => {
     message = data ?? ''
   } else {
     message = data.message ?? ''
-    logContext = extend({}, logContext, data)
-    delete logContext.message
-    delete logContext.level
-    delete logContext.timestamp
+    const { message, level, timestamp, ...dataProps } = data
```

Comment:
```suggestion
    // eslint-disable-next-line no-unused-vars
    const { message, level, timestamp, ...dataProps } = data
```

Just add this, I get an error in vs-code